dbfit-java/build.gradle

```diff
@@ -89,16 +89,19 @@ subprojects {
         classpath += sourceSets.main.runtimeClasspath
     }
 
-    ext.dbdeploy = { dbDriver, url, userid, password, delimiterType = 'normal' ->
+    ext.dbdeploy = {
+        dbDriver, url, userid, password, delimiterType = 'normal', delimiter = ';',
+        dir = "$projectDir/src/integration-test/resources" ->
```

Comment:
I meant to delete those other scripts. Yes, dbdeploy only picks up scripts that start with a number.